llvm/test/CodeGen/SPIRV/hlsl-intrinsics/dot4add_i8packed.ll

```diff
@@ -0,0 +1,65 @@
+; RUN: llc -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s --check-prefixes=CHECK,CHECK-EXP
```

Comment:
The default version will change and this test will fail. It's better to insert a version number explicitly here to be sure that it's not 1.6